src/redux/features/Plantings/plantingsSelectors.ts

```diff
@@ -9,3 +11,25 @@ export const selectPlantingsDateRange = (state: RootState, dateRange: string[])
     }
     return planting.createdTime > dateRange[0] && (dateRange.length < 2 || planting.createdTime < dateRange[1]);
   }) ?? [];
+
+export const selectPlantingProgressSubzones = createSelector(
+  [
+    (state: RootState, plantingSiteId: number) => selectPlantingSite(state, plantingSiteId),
+    (state: RootState, plantingSiteId: number) => selectPlantings(state),
+  ],
+  (plantingSite, allPlantings) => {
+    plantingSite?.plantingZones?.flatMap((zone) => {
+      return zone.plantingSubzones.map((sz) => {
+        return {
+          subzoneName: sz.fullName,
+          plantingComplete: sz.plantingCompleted,
+          plantingSite: plantingSite.name,
+          zone: zone.name,
+          targetPlantingDensity: zone.targetPlantingDensity,
+          totalSeedlingsSent: allPlantings?.find((planting) => planting.plantingSubzone.id === sz.id.toString())
```

Comment:
will this this be an expensive operation to run for all planting subzones and also for all planting sites? if so, should we scan once and create an object of { subzoneid: totalplants } as a lookup?

Reply:
we may have multiple plantings here for a subzone .. we should sum the totalPlants